tests/build/build_templates.go

```diff
@@ -33,7 +33,7 @@ import (
 
 var (
 	ecPipelineRunTimeout     = time.Duration(10 * time.Minute)
-	chainsAttestationTimeout = time.Duration(10 * time.Minute)
+	chainsAttestationTimeout = time.Duration(30 * time.Minute)
```

Comment:
Will it be good to add a comment like  `increased it to 30 min from 10 min due to https://issues.redhat.com/browse/KFLUXBUGS-24` similar to [this one](https://github.com/redhat-appstudio/e2e-tests/blob/086ef132c04731194a065313a083343f70263c27/tests/build/build.go#L281),  so that when the bug is fixed we may revert to the previous value.

Reply:
Ack, added